cogs/Voting.py

```diff
@@ -0,0 +1,421 @@
+#!/usr/bin/env python
+
+"""
+Koala Bot Base Cog code and additional base cog functions
+Commented using reStructuredText (reST)
+"""
+# Futures
+
+# Built-in/Generic Imports
+import asyncio
+import time
+
+from dotenv import load_dotenv
+
+# Libs
+import discord
+from discord.ext import commands
+
+# Own modules
+import KoalaBot
+from utils import KoalaDBManager
+
+# Constants
+load_dotenv()
+emote_reference = {1: "1️⃣", 2: "2️⃣", 3: "3️⃣",
+                   4: "4️⃣", 5: "5️⃣", 6: "6️⃣",
+                   7: "7️⃣", 8: "8️⃣", 9: "9️⃣", 10: "🔟"}
+reverse_reference = {v: k for k, v in emote_reference.items()}
+
+
+# Variables
+
+
+def is_vote_caller():
+    async def predicate(ctx):
+        cog = ctx.command.cog
+        if KoalaBot.is_dm_channel(ctx):
+            return False
+        return ctx.author.id in cog.vote_manager.active_votes.keys() and cog.vote_manager.active_votes[
+            ctx.author.id].target_server == ctx.guild.id
+
+    return commands.check(predicate)
+
+
+def test_is_admin(ctx):
+    return KoalaBot.is_admin(ctx) or ctx.author.id == 135496683009081345
+
+
+class Voting(commands.Cog):
```

Comment:
Worth using "Vote" instead of "Voting" for client interaction, just makes it a bit shorter and easier to read
```suggestion
class Voting(commands.Cog, name="Vote"):
```